src/core/listener.h

```diff
@@ -115,6 +115,11 @@ typedef struct QUIC_LISTENER {
     //
     uint8_t CibirId[2 + QUIC_MAX_CIBIR_LENGTH];
 
+    //
+    // Whether or not this listener accepts QTIP traffic (BOOLEAN)
+    //
+    BOOLEAN UseQTIP : 1;
```

Comment:
nit: `AcceptQtip`, given the comment above?

Reply:
Will be removing this field in the next iteration. 
This was a change in the earlier iterations of this draft PR.
The decision was made such that listeners always accepted both QTIP/UDP traffic.